app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -328,16 +328,18 @@ def destroy
     desc "Remove content from a repository"
     param :id, :number, :required => true, :desc => "repository ID"
     param 'ids', Array, :required => true, :desc => "Array of content ids to remove"
+    param :content_type, RepositoryTypeManager.find("yum").user_removable_content_types.map(&:label), :required => false, :desc => N_("type of repo (either 'rpm'(default), 'srpm')")
```

Comment:
actually i think we want all content_types here that are user removable, not just 'yum' ones